communicator.go

```diff
@@ -79,7 +80,7 @@ func (c *Communicator) Connect(ctx context.Context) (err error) {
 	if c.config.KeepaliveEnabled() {
 		c.logger.Println("Starting ssh KeepAlives", "host", c.host)
 		c.keepaliveDone = make(chan struct{})
-		go StartKeepalive(client, c.config.ServerAliveInterval, c.config.ServerAliveCountMax, c.keepaliveDone)
+		go KeepAlive(client, c.config.ServerAliveInterval, c.config.ServerAliveCountMax, c.keepaliveDone)
```

Comment:
Why?

Reply:
I thought that KeepAlive is self-sufficient to describe what type of action it implies

Reply:
We can rename that in a separate commit